packages/data-service/src/data-service.ts

```diff
@@ -2035,7 +2161,6 @@ class DataServiceImpl extends WithLogContext implements DataService {
       extent_last_size: data.lastExtentSize,
       flags_user: data.userFlags,
       max_document_size: data.maxSize,
-      size: data.size,
```

Comment:
Maybe I miss something but looks like this property is a duplicate of `document_size: data.size`. Correct me please if I am wrong.

Reply:
I'd double-check what we are using in the code where this method is used just in case